lib/browser/guest-window-manager.js

```diff
@@ -289,8 +289,8 @@ const handleMessageSync = function (channel, handler) {
   ipcMainUtils.handleSync(channel, makeSafeHandler(handler))
 }
 
-const assertCanAccessWindow = function (contents, guestContents) {
-  if (!canAccessWindow(contents, guestContents)) {
+const securityCheck = function (contents, guestContents, check) {
```

Comment:
Rather than passing the function in, maybe:

```
const failAccessCheck = function(senderContents, receiverContents) {
  console.error(`Blocked ${contents.getURL()} from accessing guestId: ${guestContents.id}`)
  throw new Error(`Access denied to guestId: ${guestContents.id}`)
}
```

and then the usage can be
```
if (!myCheck(...)) {
  failAccessCheck(...)
}
```
which is clearer, IMO.

Reply:
maybe, but I need to pass the `event.sender` and `guestContents` arguments to both functions now